app/views/core_induction_programmes/years/_module_list.html.erb

```diff
@@ -28,6 +28,7 @@
         </div>
 
         <% if current_user.early_career_teacher? %>
+          <hr class="govuk-section-break govuk-section-break--visible" />
```

Comment:
So we also change the display on year page? :)

Reply:
Yeah, woops, should have mentioned that in the above